larq_compute_engine/mlir/tests/lce_ops_options_test.cc

```diff
@@ -60,9 +60,9 @@ TEST(LCEOpsSerializationTest, BConv2dTest) {
   ASSERT_EQ(m["stride_height"].AsInt32(), 1);
   ASSERT_EQ(m["stride_width"].AsInt32(), 2);
   ASSERT_EQ(m["pad_values"].AsInt32(), 1);
-  ASSERT_EQ((ActivationFunctionType)m["fused_activation_function"].AsInt32(),
-            ActivationFunctionType_RELU);
-  ASSERT_EQ((Padding)m["padding"].AsInt32(), Padding_SAME);
+  ASSERT_EQ((::ActivationFunctionType)m["fused_activation_function"].AsInt32(),
+            ::ActivationFunctionType::RELU);
+  ASSERT_EQ((::Padding)m["padding"].AsInt32(), ::Padding::SAME);
```

Comment:
Would this, require changes to the kernel implementation, or are these enums identical?
@Tombana Do you see any problems with this? I guess since the test suite passes, it is fine?

Reply:
I guess the enums must be identical, but I don't fully see how that is guaranteed. But if this would ever be broken then this would definitely be caught by tests, so I think this is fine :+1:

Reply:
The enums haven't been modified, but had to add this since they were clashing with similarly named ones defined by TensorFlow.

Reply:
Sounds good, thanks for the explanation.